src/app/tests/suites/certification/Test_TC_LVL_3_1.yaml

```diff
@@ -60,6 +60,14 @@ tests:
       arguments:
           value: 0
 
+    #NOTE - fix/workaround for #2176
```

Comment:
Please use the full url, since this is an issue in a different repo.

Reply:
Updated.